tests/files/test_file.py

```diff
@@ -137,12 +137,15 @@ def test_create_from_dataframe(
     ids=["local", "s3", "gcs"],
 )
 @pytest.mark.parametrize("filetype", SUPPORTED_FILE_TYPES)
-@patch("astro.files.base.smart_open.open")
-def test_export_to_dataframe(
-    mocked_smart_open, filetype, locations, type_method_map_fixture
-):
+def test_export_to_dataframe(filetype, locations, type_method_map_fixture):
     """Test export_to_dataframe() for all locations and filetypes"""
-    with patch(type_method_map_fixture[FileType(filetype)]) as mocked_read:
+    if filetype == "parquet":
+        data = str.encode("data")
+    else:
+        data = "data"
+    with patch(type_method_map_fixture[FileType(filetype)]) as mocked_read, patch(
+        "astro.files.base.smart_open.open", mock_open(read_data=data)
+    ) as mocked_smart_open:
```

Comment:
What is the benefit of declaring the patches within the test instead of using the decorator? It would be great if we were consistent with other tests.

Reply:
@tatiana from what I can tell `mock_open` must be used as part of the context statement. When I tried adding `mock_open` to the decorator it didn't work. I can try again.

Reply:
@tatiana is that ok? :)

Reply:
Hey @dimberman if the value of read_data didn't change, we could do something like this:
`@patch( "astro.files.base.smart_open.open", new_callable=mock_open, read_data="data")`
But since it changes, I believe it would add much complexity to have this fixture dependent on the filetype! So it's okay, thank you!